### ISSUE TITLE
Are you sure? Regarding DLP with External Access only(?)

### ISSUE THREAD

**Issue by AlexHill90** [MicrosoftDocs/microsoft-365-docs#511]
"DLP for Microsoft Teams blocks sensitive content when shared with Microsoft Teams users who have:
- guest access in teams and channels; or
- external access in meetings and chat sessions.

DLP for external chat sessions will only work if both the sender and the receiver are in Teams Only mode and using Microsoft Teams native federation. DLP for Teams does not block messages in interop with Skype for Business or non-native federated chat sessions."

Based on testing, we've seen that a message can be 'marked' as DLP even in chats with internal employees, not only external. However, we're not sure on actual 'block' of the message, or if it's just 'marked' as we did test rapidly. 

I feel like we should reword the phrasing and showcase exact configurations and possibilities with DLP.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ebaab3f7-d1d4-c649-efe8-f24913da4f3e
* Version Independent ID: c5710ebf-0ca5-ab46-c490-64ebc308defe
* Content: [Data loss prevention and Microsoft Teams](https://docs.microsoft.com/en-us/microsoft-365/compliance/dlp-microsoft-teams#feedback)
* Content Source: [microsoft-365/compliance/dlp-microsoft-teams.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/dlp-microsoft-teams.md)
* Service: **o365-seccomp**
* GitHub Login: @chrfox
* Microsoft Alias: **chrfox**

**Comment by mypil** [comment]
@AlexHill90  - Thank you for submitting feedback.

I will get this issue over to the Microsoft 365 writing team for investigation.

Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.

**Comment by joinimran** [comment]
@AlexHill90 Thank you for your feedback here. I feel like the note mentioned in the document has all the things clearly mentioned. Additionally, the examples defined the scenario are clear as well. 

Regards,
Imran

**Comment by AlexHill90** [comment]
Well no, it doesn't say if the DLP works in internal teams chat, not - just - external. We're not being really 'open' in how the doc is worded on this.

**Comment by joinimran** [comment]
Can you please share where do what exact information needs to be added in the document. Let me know so that I can do that and get that verified by doc owner.

**Comment by AlexHill90** [comment]
Hi Imran, simply, I have never heard of DLP on Teams working on blocking only when talking with externals, hence the confusion.

If it is like this, it should be clearer in our messaging also in our documentation and internal references for the field.

[1200px-Microsoft_Office_Teams_(2018–present)]
Alex Collina
Technology Sales – Microsoft Teams
📱  +39 349 7727730
☎  +39 02 70392253
[Linkedin]: linkedin.com/in/alexcollina<https://www.linkedin.com/in/alexcollina>
📫  Viale Pasubio 21,20154 Milano (MI)

**Comment by joinimran** [comment]
Ok, so you are actually saying that DLP works with internal employees where as it should be working on external. But you haven't experienced DLP working on external chats. This is what I am understanding. 

Sorry there things are getting confusing for me as well. 

![image](https://user-images.githubusercontent.com/47118050/66897428-6c61c300-f010-11e9-9394-b9651d4cfad2.png)

![image](https://user-images.githubusercontent.com/47118050/66897436-708de080-f010-11e9-8fd5-67ba28bcb2c4.png)

**Comment by AlexHill90** [comment]
No, it’s the opposite. In marketing and etc. we never specified it would have worked only with external users, hence the confusion. I guess the documentation, after a couple of tests I did, is right and doesn’t need any amendment.

Our marketing should, though 😊

[1200px-Microsoft_Office_Teams_(2018–present)]
Alex Collina
Technology Sales – Microsoft Teams
📱  +39 349 7727730
☎  +39 02 70392253
[Linkedin]: linkedin.com/in/alexcollina<https://www.linkedin.com/in/alexcollina>
📫  Viale Pasubio 21,20154 Milano (MI)
